handbook/engineering/releases/patch_release_issue_template.md

```diff
@@ -17,7 +17,7 @@ Arguments:
     git log v$MAJOR.$MINOR.$(($PATCH-1))...$MAJOR.$MINOR --pretty=format:'- [ ] %H %s' --reverse
     ```
 
-## Release sourcegraph/server
+## Release Sourcegraph
 
 - [ ] Push the branch [`$MAJOR.$MINOR`](https://github.com/sourcegraph/sourcegraph/tree/$MAJOR.$MINOR) with your cherry-picked commit(s) and make sure CI passes.
 - [ ] Push a release candidate tag:
```

Comment:
TODO: see if we can use `N=<release-candidate-number> yarn run release release:create-candidate` here

Reply:
yes